lunavl/sdk/detectors/facedetector.py

```diff
@@ -307,6 +299,31 @@ def redetectOne(  # noqa: F811
             return FaceDetection(detectRes, image)
         return None
 
+    def _validateBatchReDetectInput(self, coreImages: List[CoreImage], detectAreas: List[List[Detection]]):
+        if isinstance(coreImages, list):
+            validationError, imagesErrors = self._detector.validate(coreImages, detectAreas)
+        else:
+            validationError, imagesErrors = self._detector.validate([coreImages], [[detectAreas]])
+        if validationError.isOk:
+            return
+        if validationError.error != FSDKError.ValidationFailed:
+            raise LunaSDKException(LunaVLError.fromSDKError(validationError), imagesErrors)
+        if not isinstance(coreImages, list):
+            raise LunaSDKException(LunaVLError.fromSDKError(imagesErrors[0]))
+        errors = []
+
+        for imageErrors in imagesErrors:
```

Comment:
looks the as for detectors/base.py errors processing...dry?

Reply:
for human detector has not validation function